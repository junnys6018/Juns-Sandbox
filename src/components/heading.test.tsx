import { render } from '@testing-library/react';
import Heading from './heading';

test('<Heading /> snapshot', () => {
    let { asFragment } = render(<Heading month={1} />);
    expect(asFragment()).toMatchSnapshot();

    ({ asFragment } = render(<Heading title="Foo" subTitle="bar" />));
    expect(asFragment()).toMatchSnapshot();
});
