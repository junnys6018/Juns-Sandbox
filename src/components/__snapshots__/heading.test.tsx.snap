// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Heading /> snapshot 1`] = `
<DocumentFragment>
  <h1
    class="font-semibold text-4xl text-pink-500 mb-4 md:mt-8"
  >
    January - "2.5D"
  </h1>
  <h2
    class="font-semibold text-2xl text-neutral-700"
  >
    Voxel space rendering
  </h2>
</DocumentFragment>
`;

exports[`<Heading /> snapshot 2`] = `
<DocumentFragment>
  <h1
    class="font-semibold text-4xl text-pink-500 mb-4 md:mt-8"
  >
    Foo
  </h1>
  <h2
    class="font-semibold text-2xl text-neutral-700"
  >
    bar
  </h2>
</DocumentFragment>
`;
